bool enable_motor 
bool enable_servo 
bool enable_infra 
bool enable_rele 
bool enable_rgb 
uint32 servo_pwm